src/rules/cognitive-complexity.ts

```diff
@@ -17,7 +17,6 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// https://jira.sonarsource.com/browse/RSPEC-3776
```

Comment:
tbh I didn't expect that whole line will be dropped. I agree that those Jira links should be removed, but I believe it still makes sense to keep sonar rule key somewhere. That might be useful to keep rule descriptions in sync in future and also to find quickly the related rule in sonar.

Reply:
Should just we replace these links with those from the new RSPEC repository instead ?

Reply:
sounds good to me